### DISCUSSION TITLE
missing globals in [node]: Event, EventTarget

### DISCUSSION THREAD

**Discussion by SimonSiefke** [DefinitelyTyped/DefinitelyTyped#59001] [Issues with a @types package]
These globals have been added in Node v15: https://nodejs.org/api/globals.html#event:
- Event
- EventTarget

[a comment by an automated account is not shown]

**Comment by thw0rted** [comment]
You may be interested in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59905 where I am adding the interface to a fictional module to avoid conflicts with the built-in DOM lib.  (It should be global but I'm honestly not sure how to do that without exploding every TypeScript project in a 20-mile radius...)
